Generating a fix for an incident from IDE fails

Model used: 
```
provider = "IBMOpenSource"
args = { model_id = "mistralai/mixtral-8x7b-instruct-v0-1" }
```

Workflow: 
Followed Step 1 and step 2 from https://docs.google.com/document/d/1E2e6hAbrqQNstUuqGHi49F6t2ewYj9iKXPOknrncR6g/edit#heading=h.vwokobjdwi8r 

When I clicked on an incident -> generate Kai-fix, the server failed with the following response

```
INFO - 2024-04-08 15:42:21,143 - [server.py:275 - get_incident_solution() ] - START - App: 'eap-coolstore-monolith', File: '/Users/sraghuna/local_dev/eap-coolstore-monolith/src/main/java/com/redhat/coolstore/service/InventoryNotificationMDB.java' 'kai/quarkus'/'jms-to-reactive-quarkus-00050' @ Line Number '8' using model_id 'mistralai/mixtral-8x7b-instruct-v0-1'
[2024-04-08 15:42:21 -0400] [81340] [ERROR] Error handling request
Traceback (most recent call last):
  File "/Users/sraghuna/local_dev/kai/env/lib/python3.11/site-packages/aiohttp/web_protocol.py", line 433, in _handle_request
    resp = await request_handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/env/lib/python3.11/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/server.py", line 384, in post_get_incident_solution
    llm_output = get_incident_solution(request.app, await request.json(), False).content
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/server.py", line 308, in get_incident_solution
    prompt = build_prompt(
             ^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 408, in build_prompt
    result = pb_eval(step, global_env, pb_vars)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 358, in pb_eval
    return func(fn_args, pb_env, pb_vars)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 149, in pb_concat
    arg = pb_eval(arg, pb_env, pb_vars)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 358, in pb_eval
    return func(fn_args, pb_env, pb_vars)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 254, in pb_build
    result = pb_eval(s, pb_env, pb_vars)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 358, in pb_eval
    return func(fn_args, pb_env, pb_vars)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 149, in pb_concat
    arg = pb_eval(arg, pb_env, pb_vars)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 358, in pb_eval
    return func(fn_args, pb_env, pb_vars)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 254, in pb_build
    result = pb_eval(s, pb_env, pb_vars)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 358, in pb_eval
    return func(fn_args, pb_env, pb_vars)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 190, in pb_format
    return section.format_with_raise(pb_vars)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sraghuna/local_dev/kai/kai/prompt_builder.py", line 94, in format_with_raise
    raise PBError(
kai.prompt_builder.PBError: Missing variables: {'src_file_language'} for section '$mf_input_file'
```

When I select `Kai-Fix All`, I dont see the above issue.

--- a/kai/server.py
+++ b/kai/server.py
@@ -308,7 +308,7 @@ def get_incident_solution(request_app, request_json: dict, stream: bool = False)
         pb_vars["solved_example_file_name"] = solved_incident["incident_uri"]
 
     prompt = build_prompt(
-        request_app["model_provider"].get_prompt_builder_config(), pb_vars
+        request_app["model_provider"].get_prompt_builder_config("single_file"), pb_vars
     )
     capture.prompt = prompt
 

